src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const copy = [];
+  let copyObj = { ...state };
+
+  for (const action of actions) {
+    const type = action.type;
```

Comment:
```suggestion
    const {
      type, 
      extraData, 
      keysToRemove, 
    } = action;
```
https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0